river/compat/pytorch.py

```diff
@@ -43,6 +46,23 @@ def build_torch_linear_regressor(n_features):
             "optimizer_fn": torch.optim.SGD,
         }
 
+    @classmethod
+    def _unit_test_skips(self):
+        """Indicates which checks to skip during unit testing.
+
+        Most estimators pass the full test suite. However, in some cases, some estimators might not
+        be able to pass certain checks.
+
+        """
```

Comment:
Weren't these tests working before? 

Meta: you don't need to include the comment docstring for inherited methods.